lib/xero-oauth2/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module XeroOauth2
-    VERSION = '1.0.1'
+    VERSION = '1.0.2'
```

Comment:
With `1.0.2` already published to [ruby-gems](https://rubygems.org/gems/omniauth-xero-oauth2), reckon we bump this to `1.0.3`?

Reply:
👍 have pulled into here https://github.com/XeroAPI/xero-oauth2-omniauth-strategy/pull/18